content/contributing/documentation/rst_guidelines.rst

```diff
@@ -1,162 +1,1365 @@
+
+Custom
+------
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - .. admonition:: Title
+
+          Customize this alert block with a **Title** of your choice.
+
+   * - .. code-block:: text
+
+          .. admonition:: Title
+
+             Customize this alert block with a **Title** of your choice.
+
+.. _contributing-rst-tables:
+
+Tables
+======
+
+List tables
+-----------
+
+List tables use two-level bulleted lists to convert data into a table. The first level represents
+the rows and the second level represents the columns.
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - .. list-table::
+          :header-rows: 1
+          :stub-columns: 1
+
+          * - Name
+            - Country
+            - Favorite color
+          * - Raúl
+            - Montenegro
+            - Purple
+          * - Mélanie
+            - France
+            - Red
+
+   * - .. code-block:: text
+
+          .. list-table::
+             :header-rows: 1
+             :stub-columns: 1
+
+             * - Name
+               - Country
+               - Favorite colour
+             * - Raúl
+               - Montenegro
+               - Purple
+             * - Mélanie
+               - France
+               - Turquoise
+
+Grid tables
+-----------
+
+Grid tables represent the rendered table and are more visual to work with.
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - +-----------------------+--------------+---------------+
+       |                       | Shirts       | T-shirts      |
+       +=======================+==============+===============+
+       | **Available colours** | Purple       | Green         |
+       |                       +--------------+---------------+
+       |                       | Turquoise    | Orange        |
+       +-----------------------+--------------+---------------+
+       | **Sleeves length**    | Long sleeves | Short sleeves |
+       +-----------------------+--------------+---------------+
+
+   * - .. code-block:: text
+
+          +-----------------------+--------------+---------------+
+          |                       | Shirts       | T-shirts      |
+          +=======================+==============+===============+
+          | **Available colours** | Purple       | Green         |
+          |                       +--------------+---------------+
+          |                       | Turquoise    | Orange        |
+          +-----------------------+--------------+---------------+
+          | **Sleeves length**    | Long sleeves | Short sleeves |
+          +-----------------------+--------------+---------------+
+
+.. tip::
+   - Use `=` instead of `-` to define header rows.
+   - Remove `-` and `|` separators to merge cells.
+   - Make use of `this convenient table generator <https://www.tablesgenerator.com/text_tables>`_ to
+     build your tables. Then, copy-paste the generated formatting into your document.
+
+.. _contributing-rst-code-blocks:
+
+Code blocks
```

Comment:
What is the effect of using "python" versus any other language?

Reply:
This is explained later in the code tabs section, but it would be good to include the information here as it's the first time the concept is presented:

> Just as for the code-block markup, the language defines the syntax highlighting of the tab.